app/helpers/link_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 # Helper for creating links
 module LinkHelper
-  def link_to_new_tab(name, url, **)
-    link_to(name, url, target: '_blank', rel: 'noopener', **)
+  def link_to_new_tab(*, **, &)
+    link_to(*, target: '_blank', rel: 'noopener', **, &)
   end
```

Comment:
We want our `link_to_new_tab` helper to mimic Rails `link_to` with all its warts, so this is what we need to support block-ful `link_to` invocations.